### ISSUE TITLE
Meson: Can't install redhad-sysv init style

### ISSUE THREAD

**Issue by rdmark** [Netatalk/netatalk#798]
When configuring with Meson and `-Dwith-init-style=redhat-sysv` you get an error about wrong usage of chkconfig.

```
...
Installing symlink pointing to uams_dhx2_pam.so to /usr/local/lib64/netatalk/uams_dhx2.so
Running custom install script '/usr/sbin/chkconfig add /etc/rc.d/init.d/netatalk'
chkconfig version 1.26 - Copyright (C) 1997-2000 Red Hat, Inc.
This may be freely redistributed under the terms of the GNU Public License.

usage:   chkconfig [--list] [--type <type>] [name]
         chkconfig --add <name>
         chkconfig --del <name>
         chkconfig --override <name>
         chkconfig [--level <levels>] [--type <type>] <name> <on|off|reset|resetpriorities>
FAILED: install script '/usr/sbin/chkconfig add /etc/rc.d/init.d/netatalk' exit code 1, stopped
FAILED: meson-internal__install 
/usr/bin/meson install --no-rebuild
ninja: build stopped: subcommand failed.
```

**Comment by rdmark** [comment]
This was the fix

```
diff --git a/distrib/initscripts/meson.build b/distrib/initscripts/meson.build
index 3dea5558..b653431f 100644
--- a/distrib/initscripts/meson.build
+++ b/distrib/initscripts/meson.build
@@ -14,7 +14,7 @@ elif get_option('with-init-style') == 'redhat-sysv'
     )
     meson.add_install_script(
         find_program('chkconfig'),
-        'add',
+        '--add',
         '/etc/rc.d/init.d/netatalk',
     )
 elif (
```

**Comment by dgsga** [comment]
Fix moved to PR #800
